apps/condo/domains/common/utils/filters.utils.ts

```diff
@@ -174,6 +174,10 @@ export function getFilterDropdownByKey <T> (filterMetas: Array<FiltersMeta<T>>,
     }
 }
 
+export function convertToOptions <T> (objects: T[], labelField: string, valueField: string): OptionType[] {
+    return objects.map(object => ({ label: object[labelField], value: object[valueField] }))
```

Comment:
if `object` is  not an `Object` really, it will throw here. so please use `get(object, labelField)` instead

Reply:
fixed in 8b91092